src/components/Results/__tests__/Results.test.tsx

```diff
@@ -1,14 +1,62 @@
 import { render, screen } from '@testing-library/react';
-import { Bundle } from 'fhir/r4';
+import userEvent from '@testing-library/user-event';
+import { ResearchStudy } from 'fhir/r4';
 import Results, { ResultsProps } from '../Results';
-import mockSearchResults from '@/__mocks__/results.json';
+import mockStudies from '@/__mocks__/studies.json';
+
+afterEach(() => {
+  jest.clearAllMocks();
```

Comment:
This `jest.clearAllMocks()` is needed because the internal counter tracked by `toHaveBeenCalledTimes()` will carry over from each test.